dist.ini

```diff
@@ -0,0 +1,44 @@
+name = AnyEvent-RabbitMQ
+author = Masahito Ikuta <cooldaemon@gmail.com>
+license = Perl_5
+copyright_holder = AnyEvent-RabbitMQ's developers
+
+[@Filter]
+-bundle = @Basic
+-remove = Readme
+
+[Metadata]
+x_contributors = Tom Doran <bobtfish@cpan.org>
+x_contributors = Nicolas R <atoomic@cpan.org>
+x_contributors = Dave Lambley <dlambley@cpan.org>
+x_contributors = Ruslan Zakirov <ruz@bestpractical.com>
+x_contributors = Masahito Ikuta <cooldaemon@gmail.com>
+x_contributors = Rod Taylor <rod.taylor@gmail.com>
+x_contributors = Carl Hörberg <carl@cloudamqp.com>
+
+[InstallGuide]
+;[CPANFile]
+[VersionFromModule]
+[MetaJSON]
+
+[Repository]
+repository = git://github.com/bobtfish/AnyEvent-RabbitMQ.git
+
+[Prereqs]
+List::MoreUtils = 0
+Net::AMQP = 0.06
+AnyEvent = 0
+Devel::GlobalDestruction = 0
+namespace::clean = 0
+File::ShareDir = 0
+Readonly = 1.03
+
+[Prereqs / TestRequires]
+Test::More = 0
+Test::Exception = 0
+version = 0
+
+[Prereqs / TestRecommends]
+Test::Spelling = 0
+Test::Perl::Critic = 0
+Test::Pod = 0
```

Comment:
I suggest to use various Dist::Zilla::Plugin::Test plugins which add their requirements and also generate the test files.